stream_alert/rule_processor/handler.py

```diff
@@ -27,14 +28,17 @@
 from stream_alert.rule_processor.sink import StreamSink
 from stream_alert.shared.metrics import MetricLogger
 
-
+MAX_BATCH_COUNT = 500
+MAX_BATCH_SIZE = 4000 * 1000
 MAX_RECORD_SIZE = 1000 * 1000
-MAX_BATCH_SIZE = 500
 
 
 class StreamAlert(object):
     """Wrapper class for handling StreamAlert classificaiton and processing"""
-    __config = {}
+    config = {}
+    special_char_regex = re.compile(r'[\`\~\!\@\#\$\%\^\&\*\(\)\-\+\
+                                       \=\[\]\\\{\}\;\'\:\"\,\.\/\<\>\?]')
```

Comment:
Instead of trying to list every possible special character, `\W` matches any non-alphanumeric character (i.e. it is equivalent to `[^a-zA-Z0-9_]`). This should work quite nicely for you